### PR TITLE
Update google-drive from 46.0.3.0 to 47.0.19 and add livecheck

### PR DESCRIPTION
**Pull request by cho-m**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

---

Try using `:extract_plist` for version.
Major/Minor version (47.0) at least matches the release notes: https://support.google.com/a/answer/7577057

May want to get a user to check if auto-updates are supported since most Google Casks do have stanza set.
From support page:
- https://support.google.com/a/answer/7491144#zippy=%2Cmac
   > Drive for desktop comes packaged with Google Update (Windows) or Google Software Update (Mac) to auto-update Drive for desktop on your users’ computers. 
- https://support.google.com/a/answer/10514878
   > When auto-updates is on, users always have the latest version. Your users might install and set up Google Drive for desktop themselves. However, only admins can configure auto-update settings.

### CODE REVIEW

File: Casks/google-drive.rb [hunk 1]
```diff
@@ -22,21 +23,22 @@
...
               "com.google.drivefs.filesystems.dfsfuse.x86_64",
               "com.google.drivefs.shortcuts",
+              "com.google.pkg.Keystone",
+            ],
+            launchctl:  [
+              "com.google.keystone.agent",
+              "com.google.keystone.system.agent",
+              "com.google.keystone.daemon",
+              "com.google.keystone.xpcservice",
+              "com.google.keystone.system.xpcservice",
```
Comment (miccal) [inline Casks/google-drive.rb:33]: Please revert these changes and leave them in the `zap` stanza -- these are used by _a lot_ of Google software, and removing them during and uninstall/upgrade is an issue.

Comment (cho-m) [reply Casks/google-drive.rb:33]: Sure.
The CI was failing due to these leftover, so will need to manually bypass the failure.

Comment (miccal) [reply Casks/google-drive.rb:33]: Yes, that is standard for `google-*` Casks.

**Review by miccal — APPROVED** [review]
